.github/CONTRIBUTING.md

```diff
@@ -2,6 +2,37 @@
 
 Thank you for your interest in contributing to Octicons! We are currently only accepting submissions from GitHub staff and only include icons that are used in the GitHub UI.
 
+## Icon request process
+
+### 1. Icon request is made
+
+- Icon requests are made using the issue template in the primer/octicons repo. If the icon being requested is for a product/feature that has not yet been release, please use the [template](https://github.com/github/design-systems/issues/new?assignees=ashygee%2C+edokoa&labels=area%3A+icons%2C+type%3A+request&template=octicon_request.md&title=%5BIcon+request%5D+) found in the [github/design-systems repo](https://github.com/github/design-systems/issues/new/choose).
```

Comment:
It'd be nice to mention that we don't accept icon request from outside of GitHub here.

Reply:
@colebemis that's mentioned here https://github.com/primer/octicons/pull/547/files#diff-98e64bc1cd2db9333c6effe87bbe0d6dfe8714aba4c6bde45aa037fe0796e44cR3

Reply:
Oh duh 🤦